spec/deployment_spec.rb

```diff
@@ -0,0 +1,227 @@
+# frozen_string_literal: true
+#
+# Author:: Chef Partner Engineering (<partnereng@chef.io>)
```

Comment:
author could be updated in the new specs.

Reply:
Done